chpl/chpl-auth/src/main/java/gov/healthit/chpl/auth/entity/UserResetTokenEntity.java

```diff
@@ -0,0 +1,115 @@
+package gov.healthit.chpl.auth.entity;
+
+import java.util.Date;
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import gov.healthit.chpl.auth.user.User;
+
+
+@Entity
+@Table(name = "user_reset_token")
+public class UserResetTokenEntity {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Does any other class have the UID of 1? Seems like a potential conflict... it's usually a weird randomly generated long value.

Reply:
I dont know why Github still shows this code but I did update the entity to not have a serialUID because most others dont